lib/blacklight/access_controls/enforcement.rb

```diff
@@ -25,72 +29,61 @@ module Enforcement
 
       delegate :current_ability, to: :scope
 
-      protected
-
-      def gated_discovery_filters(permission_types = discovery_permissions, ability = current_ability)
-        user_access_filters = []
-
-        # Grant access based on user id & group
-        solr_access_filters_logic.each do |method_name|
-          user_access_filters += send(method_name, permission_types, ability)
-        end
-        user_access_filters
-      end
-
-      #
-      # Solr query modifications
-      #
-
-      # Set solr_parameters to enforce appropriate permissions
-      # * Applies a lucene query to the solr :q parameter for gated discovery
-      # * Uses public_qt search handler if user does not have "read" permissions
-      # @param solr_parameters the current solr parameters
-      def add_access_controls_to_solr_params(solr_parameters)
-        apply_gated_discovery(solr_parameters)
-      end
-
       # Which permission levels (logical OR) will grant you the ability to discover documents in a search.
-      # Override this method if you want it to be something other than the default
+      # Override this method if you want it to be something other than the default, or hit the setter
```

Comment:
This is below the level of PR review I regard as substantive.  The comment is *improved* by the addition.  It can be merged **as is** without detriment.  If you think it could be better still, you are probably correct!  But you are also welcome to submit further revisions, including against this branch.  IMO, the value represented by this PR is pretty well separate from the difference between the improved comment and a potentially better one.